plantseg/viewer/widget/io.py

```diff
@@ -212,6 +273,31 @@ def export_stack_as_h5(data,
     return out_path
 
 
+def export_stack_as_zarr(data,
+                         name,
+                         directory,
+                         voxel_size,
+                         voxel_size_unit,
+                         custom_name,
+                         standard_suffix,
+                         scaling_factor,
+                         order,
+                         stack_type,
+                         dtype):
+    if scaling_factor is not None:
+        data = image_rescale(data, factor=scaling_factor, order=order)
+
+    key = f'export_{standard_suffix}' if custom_name is None else custom_name
+
+    directory = Path(directory)
+    directory.mkdir(parents=True, exist_ok=True)
+    out_path = directory / f'{name}.zarr'
```

Comment:
let's use `os.path.join` here as well, so we're Windows friendly :)

Reply:
This is using the `/` overload from `pathlib.Path` so it will work on Windows (and any OS), and from what I understand is the preferred and more idiomatic way to do this in recent pythons.

Unless you have strong opinions against `pathlib`, I would stick with this solution and slowly replace all `os.path.join`. PS `pathlib` is also more magicgui friendly!

Reply:
oh, so `/` is overloaded in `pathlib`, then sure let's go with `pathlib`~